### ISSUE TITLE
Convert "Annex D EXPRESS-G diagrams"

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/stepmod2mn#14]
In `resource_docs/*/resource.xml` files each schema is provided with diagram information:

e.g.
```xml
   <schema_diag>
      <express-g>
         <imgfile file="schema_diagexpg1.xml"
                  title="The schemas of this part of ISO 10303"/>
      </express-g>
   </schema_diag>
```

```xml
   <schema name="application_context_schema"
           number="10231"
           version="8"
           number.supersedes="8369">
      <introduction>
      ...
      </introduction>
      <fund_cons>
      ...
      </fund_cons>
      <express-g>
         <imgfile file="application_context_schemaexpg1.xml"/>
         <imgfile file="application_context_schemaexpg2.xml"/>
      </express-g>
   </schema>
```

These get rendered into Annex D (check the `xsl/` dir):
(see `iso-10303-smrl/reference-docs/N10480_SMRL_v8_rc5/data/resource_docs/fundamentals_of_product_description_and_support/sys/d_expg.htm`)

<img width="1258" alt="Screenshot 2021-02-25 at 11 37 02 AM" src="https://user-images.githubusercontent.com/11865/109099287-ca063380-775d-11eb-869b-70056f4ac274.png">

There is this boilerplate:
> Annex D
> (informative)
> 
> EXPRESS-G diagrams
> 
> The diagrams in this annex correspond to the EXPRESS schemas specified in this part of ISO 10303. The diagrams use the EXPRESS-G graphical notation for the EXPRESS language. EXPRESS-G is defined in ISO 10303-11.

And in particular you see that this XML:
```xml
      <express-g>
         <imgfile file="application_context_schemaexpg1.xml"/>
         <imgfile file="application_context_schemaexpg2.xml"/>
      </express-g>
```
is now:
<img width="584" alt="Screenshot 2021-02-25 at 11 37 44 AM" src="https://user-images.githubusercontent.com/11865/109099328-e1452100-775d-11eb-8976-4670dee0e096.png">

And when you click into it it shows this:
<img width="753" alt="Screenshot 2021-02-25 at 11 37 52 AM" src="https://user-images.githubusercontent.com/11865/109099345-e73b0200-775d-11eb-8028-7474c73a7e3d.png">

Basically it should be converted into an ADoc snippet of:

```adoc
[appendix,obligation=informative]
== EXPRESS-G diagrams

...

.EXPRESS-G diagram of the application_context_schema (1 of 2)
image::application_context_schemaexpg1.svg[]

.EXPRESS-G diagram of the application_context_schema (2 of 2)
image::application_context_schemaexpg2.svg[]

...
```

**Comment by Intelligent2013** [comment]
@ronaldtse Issue fixed. Now it converted into:
```adoc
[[AnnexD]]
[appendix,obligation=informative]
== EXPRESS-G diagrams

The diagrams in this annex correspond to the EXPRESS schemas specified in this part of ISO 10303. The diagrams use the EXPRESS-G graphical notation for the EXPRESS language. EXPRESS-G is defined in ISO 10303-11.

.EXPRESS-G diagram of the action_schema (1 of 2)
image::action_schemaexpg1.svg[]

.EXPRESS-G diagram of the action_schema (2 of 2)
image::action_schemaexpg2.svg[]

.EXPRESS-G diagram of the application_context_schema (1 of 2)
image::application_context_schemaexpg1.svg[]

.EXPRESS-G diagram of the application_context_schema (2 of 2)
image::application_context_schemaexpg2.svg[]

.EXPRESS-G diagram of the approval_schema (1 of 2)
image::approval_schemaexpg1.svg[]

.EXPRESS-G diagram of the approval_schema (2 of 2)
image::approval_schemaexpg2.svg[]

.EXPRESS-G diagram of the basic_attribute_schema (1 of 5)
image::basic_attribute_schemaexpg1.svg[]

.EXPRESS-G diagram of the basic_attribute_schema (2 of 5)
image::basic_attribute_schemaexpg2.svg[]

.EXPRESS-G diagram of the basic_attribute_schema (3 of 5)
image::basic_attribute_schemaexpg3.svg[]

.EXPRESS-G diagram of the basic_attribute_schema (4 of 5)
image::basic_attribute_schemaexpg4.svg[]

.EXPRESS-G diagram of the basic_attribute_schema (5 of 5)
image::basic_attribute_schemaexpg5.svg[]

.EXPRESS-G diagram of the certification_schema (1 of 2)
image::certification_schemaexpg1.svg[]

.EXPRESS-G diagram of the certification_schema (2 of 2)
image::certification_schemaexpg2.svg[]

.EXPRESS-G diagram of the contract_schema (1 of 2)
image::contract_schemaexpg1.svg[]

.EXPRESS-G diagram of the contract_schema (2 of 2)
image::contract_schemaexpg2.svg[]

.EXPRESS-G diagram of the date_time_schema (1 of 3)
image::date_time_schemaexpg1.svg[]

.EXPRESS-G diagram of the date_time_schema (2 of 3)
image::date_time_schemaexpg2.svg[]

.EXPRESS-G diagram of the date_time_schema (3 of 3)
image::date_time_schemaexpg3.svg[]

.EXPRESS-G diagram of the document_schema (1 of 2)
image::document_schemaexpg1.svg[]

.EXPRESS-G diagram of the document_schema (2 of 2)
image::document_schemaexpg2.svg[]

.EXPRESS-G diagram of the effectivity_schema (1 of 2)
image::effectivity_schemaexpg1.svg[]

.EXPRESS-G diagram of the effectivity_schema (2 of 2)
image::effectivity_schemaexpg2.svg[]

.EXPRESS-G diagram of the experience_schema (1 of 2)
image::experience_schemaexpg1.svg[]

.EXPRESS-G diagram of the experience_schema (2 of 2)
image::experience_schemaexpg2.svg[]

.EXPRESS-G diagram of the external_reference_schema (1 of 2)
image::external_reference_schemaexpg1.svg[]

.EXPRESS-G diagram of the external_reference_schema (2 of 2)
image::external_reference_schemaexpg2.svg[]

.EXPRESS-G diagram of the group_schema (1 of 2)
image::group_schemaexpg1.svg[]

.EXPRESS-G diagram of the group_schema (2 of 2)
image::group_schemaexpg2.svg[]

.EXPRESS-G diagram of the language_schema (1 of 2)
image::language_schemaexpg1.svg[]

.EXPRESS-G diagram of the language_schema (2 of 2)
image::language_schemaexpg2.svg[]

.EXPRESS-G diagram of the location_schema (1 of 2)
image::location_schemaexpg1.svg[]

.EXPRESS-G diagram of the location_schema (2 of 2)
image::location_schemaexpg2.svg[]

.EXPRESS-G diagram of the management_resources_schema (1 of 9)
image::management_resources_schemaexpg1.svg[]

.EXPRESS-G diagram of the management_resources_schema (2 of 9)
image::management_resources_schemaexpg2.svg[]

.EXPRESS-G diagram of the management_resources_schema (3 of 9)
image::management_resources_schemaexpg3.svg[]

.EXPRESS-G diagram of the management_resources_schema (4 of 9)
image::management_resources_schemaexpg4.svg[]

.EXPRESS-G diagram of the management_resources_schema (5 of 9)
image::management_resources_schemaexpg5.svg[]

.EXPRESS-G diagram of the management_resources_schema (6 of 9)
image::management_resources_schemaexpg6.svg[]

.EXPRESS-G diagram of the management_resources_schema (7 of 9)
image::management_resources_schemaexpg7.svg[]

.EXPRESS-G diagram of the management_resources_schema (8 of 9)
image::management_resources_schemaexpg8.svg[]

.EXPRESS-G diagram of the management_resources_schema (9 of 9)
image::management_resources_schemaexpg9.svg[]

.EXPRESS-G diagram of the measure_schema (1 of 5)
image::measure_schemaexpg1.svg[]

.EXPRESS-G diagram of the measure_schema (2 of 5)
image::measure_schemaexpg2.svg[]

.EXPRESS-G diagram of the measure_schema (3 of 5)
image::measure_schemaexpg3.svg[]

.EXPRESS-G diagram of the measure_schema (4 of 5)
image::measure_schemaexpg4.svg[]

.EXPRESS-G diagram of the measure_schema (5 of 5)
image::measure_schemaexpg5.svg[]

.EXPRESS-G diagram of the person_organization_schema (1 of 4)
image::person_organization_schemaexpg1.svg[]

.EXPRESS-G diagram of the person_organization_schema (2 of 4)
image::person_organization_schemaexpg2.svg[]

.EXPRESS-G diagram of the person_organization_schema (3 of 4)
image::person_organization_schemaexpg3.svg[]

.EXPRESS-G diagram of the person_organization_schema (4 of 4)
image::person_organization_schemaexpg4.svg[]

.EXPRESS-G diagram of the product_definition_schema (1 of 3)
image::product_definition_schemaexpg1.svg[]

.EXPRESS-G diagram of the product_definition_schema (2 of 3)
image::product_definition_schemaexpg2.svg[]

.EXPRESS-G diagram of the product_definition_schema (3 of 3)
image::product_definition_schemaexpg3.svg[]

.EXPRESS-G diagram of the product_property_definition_schema (1 of 2)
image::product_property_definition_schemaexpg1.svg[]

.EXPRESS-G diagram of the product_property_definition_schema (2 of 2)
image::product_property_definition_schemaexpg2.svg[]

.EXPRESS-G diagram of the product_property_representation_schema (1 of 2)
image::product_property_representation_schemaexpg1.svg[]

.EXPRESS-G diagram of the product_property_representation_schema (2 of 2)
image::product_property_representation_schemaexpg2.svg[]

.EXPRESS-G diagram of the qualifications_schema (1 of 2)
image::qualifications_schemaexpg1.svg[]

.EXPRESS-G diagram of the qualifications_schema (2 of 2)
image::qualifications_schemaexpg2.svg[]

.EXPRESS-G diagram of the security_classification_schema (1 of 2)
image::security_classification_schemaexpg1.svg[]

.EXPRESS-G diagram of the security_classification_schema (2 of 2)
image::security_classification_schemaexpg2.svg[]

.EXPRESS-G diagram of the support_resource_schema (1 of 2)
image::support_resource_schemaexpg1.svg[]

.EXPRESS-G diagram of the support_resource_schema (2 of 2)
image::support_resource_schemaexpg2.svg[]
```

**Comment by ronaldtse** [comment]
We have some challenge here -- because we don't know where the images are... hmmm

**Comment by Intelligent2013** [comment]
Images in the folder `/data/resource/<schema>`.
For example: https://github.com/metanorma/iso-10303-stepmod/blob/master/data/resources/action_schema/action_schemaexpg1.svg
(please note, that latest version of SVGs (with numbered href) in the branch `annotate7`)

How would you like to specify path:
- relation like `../../resource/action_schema/action_schemaexpg1.svg`
or
- absolute from some root like `data/resource/action_schema/action_schemaexpg1.svg`

And I think the path should be set in svgmap section:
```
[.svgmap]
====
image::../../resource/action_schema/reschema_diagexpg1.svg[]

* <<express:group_schema>>; 1
* <<express:qualifications_schema>>; 2
* <<express:location_schema>>; 3
```

**Comment by ronaldtse** [comment]
Yes it should be svgmap, not just image::.

**Comment by Intelligent2013** [comment]
So,  it's ok?

```
[.svgmap]
====
image::../../resources/action_schema/action_schemaexpg1.svg[]

* <<express:basic_attribute_schema>>; 1
* <<express:action_schema>>; 2
* <<express:support_resource_schema>>; 3
====
...


...
[[AnnexD]]
[appendix,obligation=informative]
== EXPRESS-G diagrams

The diagrams in this annex correspond to the EXPRESS schemas specified in this part of ISO 10303. The diagrams use the EXPRESS-G graphical notation for the EXPRESS language. EXPRESS-G is defined in ISO 10303-11.

.EXPRESS-G diagram of the action_schema (1 of 2)
image::action_schemaexpg1.svg[]

...
```

**Comment by opoudjis** [comment]
The image path in Annex D should be: `image::../../../../resources/action_schema/action_schemaexpg1.svg[]`

No, I do not understand why.

**Comment by Intelligent2013** [comment]
@opoudjis I.e. option 2 or 1?

Option 1 (relative path in .svgmap section):
```
[.svgmap]
====
image::../../../../resources/action_schema/action_schemaexpg1.svg[]

* <<express:basic_attribute_schema>>; 1
* <<express:action_schema>>; 2
* <<express:support_resource_schema>>; 3
====
...


...
[[AnnexD]]
[appendix,obligation=informative]
== EXPRESS-G diagrams

The diagrams in this annex correspond to the EXPRESS schemas specified in this part of ISO 10303. The diagrams use the EXPRESS-G graphical notation for the EXPRESS language. EXPRESS-G is defined in ISO 10303-11.

.EXPRESS-G diagram of the action_schema (1 of 2)
image::action_schemaexpg1.svg[]

...
```

or

Option 2 (relative path in Annex section):


```
[.svgmap]
====
image::action_schemaexpg1.svg[]

* <<express:basic_attribute_schema>>; 1
* <<express:action_schema>>; 2
* <<express:support_resource_schema>>; 3
====
...


...
[[AnnexD]]
[appendix,obligation=informative]
== EXPRESS-G diagrams

The diagrams in this annex correspond to the EXPRESS schemas specified in this part of ISO 10303. The diagrams use the EXPRESS-G graphical notation for the EXPRESS language. EXPRESS-G is defined in ISO 10303-11.

.EXPRESS-G diagram of the action_schema (1 of 2)
image::../../../../resources/action_schema/action_schemaexpg1.svg[]

...
```

**Comment by opoudjis** [comment]
Option 2

**Comment by Intelligent2013** [comment]
Done.

**Comment by ronaldtse** [comment]
@Intelligent2013 we have a change here, will file new ticket.
